test/date.js

```diff
@@ -142,9 +142,9 @@ describe('Date Component', () => {
         res.render = sinon.stub();
       });
 
-      it('calls res.render with template & key. It also calls it with legendClassName as \'\' and legend & hint as null', () => {
+      it('calls res.render with template & key.', () => {
         date.hooks['pre-render'](req, res, next);
-        expect(res.render).to.have.been.calledWith(path.resolve(__dirname, '../templates/date.html'), { key: 'date-field', legend: null, legendClassName: '', hint: null, error: undefined});
+        expect(res.render).to.have.been.calledWith(path.resolve(__dirname, '../templates/date.html'), sinon.match({key: 'date-field'}));
```

Comment:
I kept the actual template as part of the test rather than using sinon.match.string as I thought for the first test it was relevant to check the template